### ISSUE TITLE
ctrl-enter doesn't seem to work

### ISSUE THREAD

**Issue by matt-snider** [arybczak/ncmpcpp#324]
Hi there!

First of all, thank you for this great project! I've been happily using it for several months. :smile: 

Today I wanted to tweak some keyboard shortcuts and was hoping to be able to submit the search form with `ctrl-enter`. This did not work:
```
def_key "ctrl-enter"
    start_searching
```

This, however, does work:
```
def_key "alt-enter"
    start_searching
```

I'm wondering if this is by design/technical challenge or if it's a bug.

Thank you!
Matt

**Comment by arybczak** [comment]
Hey,

Terminal doesn't recognize ctrl+enter (it sees it as enter only), so it's a technical limitation.

**Comment by matt-snider** [comment]
Oh, okay - thanks for the info!

On January 16, 2019 6:21:08 PM UTC, Andrzej Rybczak <notifications@github.com> wrote:
>Closed #324.

--
Best regards, 
Matt Snider
